### DISCUSSION TITLE
Shimmer locked in Genesis

### DISCUSSION THREAD

**Discussion by Guenter6724** [iotaledger/firefly#4898] [Help & Support]
I can´t send my shimmer, cause they are locked in Shimmer Genesis in my firefly shimmer wallet
Could please someone help me?
Thanks

**Comment by maxwellmattryan** [comment]
Have you tried first claiming your Shimmer rewards? If they've been claimed (through Firefly) then they won't be "locked" in the genesis blocks anymore.
